nexus/src/app/ip_pool.rs

```diff
@@ -25,6 +27,10 @@ use omicron_common::api::external::ResourceType;
 use omicron_common::api::external::UpdateResult;
 use ref_cast::RefCast;
 
+fn is_internal(pool: &IpPool) -> bool {
```

Comment:
I wanted to, but `models` can't import from `db-queries` because `db-queries` depends on `models`, and `db-queries` is where `INTERNAL_SILO_ID` is defined.

Reply:
ah, fair enough